evaluation.md

```diff
@@ -5,15 +5,99 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
-**On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
+Our syntax is trees, named states, and naming walls and blanks rather than 
+using NEWS.  We also wanted to be able to specify the starting state rather
+than having it depend on the position of the rules within the List.
 
-**On a scale of 1–10 (where 10 is "very difficult"), how difficult was it to map your syntax to the provided API?**
+ * We were able to keep named states, since this was already implemented in the
+provided API.
+ * We kept the description of walls and blanks (i.e. `"NE! S_"` to describe 
+ walls to the North and East, a blank to the South, and anything to the West.
+ One compromise here was that the user needed to make this a string, because
+ there are too many permutations of expressing a surrounding in this way for
+ us to make objects represent surroundings.
+ * We gave up on having a tree structure for states (in which rules of each
+  state are nested together within the same block) for several reasons.
```

Comment:
It's totally possible (as Tiffany mentioned), but as we learned it's pretty messy and does take a lot of time. For ours, I know it was pretty essential to our implementation, so it made sense to implement it. However, I also understand not wanting to, especially in the internal implementation because it was a lot of effort.